docs/make.jl

```diff
@@ -13,27 +13,30 @@ format = Documenter.HTML(edit_link = "source",
 
 makedocs(modules  = [Images, ImageCore, Colors, ColorTypes, FixedPointNumbers, ImageAxes,
                     ImageFeatures, ImageFiltering, ImageMetadata, ImageContrastAdjustment,
+                    ImageDistances, ImageQualityIndexes, MosaicViews, TestImages, FileIO,
                     ImageSegmentation, ImageTransformations, PaddedViews, ImageMorphology],
          format   = format,
          sitename = "JuliaImages",
          pages    = ["Home" => "index.md",
-                     "install.md",
-                     "Manual" => Any[
-                         "quickstart.md",
-                         "arrays_colors.md",
-                         "conversions_views.md",
-                         "indexing.md",
-                         "imageaxes.md",
-                         "imagefiltering.md",
-                         "imagemetadata.md",
-                         "imagesegmentation.md",
-                         "imagetransformations.md",
-                         "imagefeatures.md",
-                         "troubleshooting.md",
+                     "Tutorials" => Any[
+                         joinpath("tutorials", "install.md"),
+                         joinpath("tutorials", "quickstart.md"),
```

Comment:
Should `install.md` and `quickstart.md` included into `Tutorials`? If the answer is yes, we might need to move https://github.com/JuliaImages/juliaimages.github.io/blob/source/docs/src/quickstart.md#L302-L342 to a better place.